### ISSUE TITLE
Sculpt update requires replacing deploy runtime

### ISSUE THREAD

**Issue by ehmry** [genodelabs/genode#2817]
Exiting VMs to update depot packages is getting to be an annoyance. Loading the system initially from a dedicated runtime is fine, but the depot should be upgradeable from the deploy runtime.

**Comment by ehmry** [comment]
I've added a runtime file and additional archives to the `depot_download` package at 83681ac277eca5e233f2acd8b147f2298fceffae. This allows updates from the deploy runtime.

My */config/deploy/config*:
```
	<start name="depot_download" pkg="ehmry/pkg/depot_download/2018-05-08" version="1">
		<route>
			<service name="Nic">    <child name="nic_router"/> </service>
			<service name="RM">     <parent/> </service>
			<service name="Rtc">    <parent/> </service>
			<service name="ROM" label_suffix="installation">
				<parent label="installation"/> </service>
			<service name="File_system">
				<parent label="depot"/> </service>
		</route>
	</start>
```

And the routing in */config/runtime/deploy.config*:
```
	<start name="dynamic" caps="30000">
		<resource name="RAM" quantum="7G"/>
		<binary name="init"/>
		<route>
			<service name="ROM" label="installation">
				<parent label="config -> installation"/> </service>
			<service name="File_system" label="depot"> <child name="fs"/> </service>
			...
		</route>
	</start>

```

**Comment by ehmry** [comment]
Fixed in 18.05.
